docs/proposals/27-project-group.md

```diff
@@ -0,0 +1,138 @@
+  taintSeed: false
+  seedRef: project1-aws-ha1
+  seedSelector:
+    region: eu-west1
+
+```
+
+If `.spec.taintSeed` is set to `true`, the specified seed(s) will be tainted, so that no other projects outside the project group can schedule shoots on them.
+
+Such tainting bindings will only be allowed to be created by Gardener operators, so that not any arbitrary user can exclusively bind seeds.
+
+Seed bindings will only work in either project or project group namespaces.
+
+As mentioned in [ProjectGroup](#projectgroup), when a seed binding is present in a project group namespace, it will be copied to the respective project namespace.
+
+It will also be possible to allow seed bindings to be in a project namespace, in which it becomes available for a singular project.
+
+The `.spec.seedRef` field specifies which Seed is bound to the project group / project.
+
+Alternatively to such a specific seed, it will be possible to select one or more seeds by a label selector defined in `.spec.seedSelector`.
+
+Either `.spec.seedRef` and `.spec.seedSelector` will only be usable exclusively.
+
+## Alternatives
+
+### Implementation without ProjectGroup
+
+Alternatively to introducing a new concept that spans above the hierarchy of projects, a mechanism could be introduced that lies on the level of projects itself, like sharing resources from one project to another.
+
+A downside of this could be that in actual use, a singular project could be used as a kind of "template" from which related projects get the relevant resources or information to schedule shoots on a private seed. As a result, this template project would have the same function as a project group, but from an architectual point of view, it would not be so clear that it is just that.
+
+Therefore this concept is not really feasible.
+
+### Resource sharing via hierarchical namespaces
+
+The [Hierarchical Namespace Controller](https://github.com/kubernetes-sigs/hierarchical-namespaces?tab=readme-ov-file) would allow to share the proposed `SeedBinding`s natively. Even though the repository still received commits fairly recently, there is no general availability yet and the latest [release]((https://github.com/kubernetes-sigs/hierarchical-namespaces/releases/tag/v1.1.0)) is quite old.
+
+In addition to that, this new dependency would require reasonable additional effort to implement into the Gardener project, so intuitively it does not seem reasonable to use this controller in this context.
+
+### Exclusively leveraging existing Taints and Tolerations
+
+It could be possible to completely rely on the existing tainting functionality of seeds and tolerations of shoots, but a problem arises, when two projects bind the same seed. This could be the case, when a user group wants to ensure e.g. that their control plane lands on a seed in a specific country. This would mean a taint for this project would be set. As all taints need to be tolerated by the shoot, a scenario like this would not be possible.
```

Comment:
As far as I understand, the main problem would be coordination among the taints/tolerations to be used by multiple projects. This is not clear from this paragraph. Please clarify.

Reply:
I included an example now in [505e4e5](https://github.com/gardener/gardener/pull/9991/commits/505e4e5acf430ea272afda4ba4d162d7de8c98b2) that only relying on the taints and tolerations feature makes it hard to manage